client/js/components/GameView/ReauthenticateDialog.vue

```diff
@@ -2,31 +2,33 @@
   <v-dialog v-model="show" persistent>
     <v-card id="reauthenticate-dialog">
       <v-card-title>Reconnect to Game</v-card-title>
-      <v-card-text>
-        <p>You have disconnected due to inactivity. Log in again to resume your session</p>
-        <!-- Login Form -->
-        <v-text-field
-          v-model="username"
-          outlined
-          :dense="$vuetify.breakpoint.mdAndDown"
-          label="Username"
-          data-cy="username"
-          @keyup.enter="login"
-        />
-        <v-text-field
-          v-model="password"
-          outlined
-          label="Password"
-          :dense="$vuetify.breakpoint.mdAndDown"
-          type="password"
-          data-cy="password"
-          @keyup.enter="login"
-        />
-      </v-card-text>
-      <v-card-actions class="d-flex justify-end">
-        <v-btn text color="primary" @click="leaveGame"> Leave Game </v-btn>
-        <v-btn color="primary" depressed data-cy="login" @click="login"> Log In </v-btn>
-      </v-card-actions>
+      <form @submit.prevent="login">
+        <v-card-text>
+          <p>You have disconnected due to inactivity. Log in again to resume your session</p>
+          <!-- Login Form -->
+          <v-text-field
+            v-model="username"
+            outlined
+            :dense="$vuetify.breakpoint.mdAndDown"
+            label="Username"
+            data-cy="username"
+            @keyup.enter="login"
+          />
+          <v-text-field
+            v-model="password"
+            outlined
+            label="Password"
+            :dense="$vuetify.breakpoint.mdAndDown"
+            type="password"
+            data-cy="password"
+            @keyup.enter="login"
```

Comment:
We no longer need the key up handler here or on the other input because hitting enter will submit the form and trigger the top level handler
```suggestion
```